jest.config.js

```diff
@@ -52,7 +52,9 @@ module.exports = {
     },
   },
   collectCoverageFrom: coverageDirs,
-  reporters: [`default`].concat(useCoverage ? `jest-junit` : []),
+  reporters: process.env.CI
+    ? [[`jest-silent-reporter`, { useDots: true }]]
```

Comment:
this is to make debugging our CI test failures easier. Right now the output is so large it takes a while for circle ci to actually render the test output. This change makes it so that we only report test failures in CI.